src/main/resources/assets/computercraft/lua/rom/apis/textutils.lua

```diff
@@ -149,10 +158,32 @@ local function tabulateCommon( bPaged, ... )
 end
 
 function tabulate( ... )
+    tArgs = { ... }
+    for i=1,#tArgs,2 do
```

Comment:
You're creating a global here, which is never good. It might be better to use `select('#', ...)` and `select(i, ...)` instead - that way it won't break on `nil`s.